interface/backend/api/views.py

```diff
@@ -42,6 +46,78 @@ class ImageSeriesViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.ImageSeriesSerializer
 
 
+class ImageMetadataApiView(APIView):   
+
+    def _sanitise_unicode(self, s):
+        return s.replace(u"\u0000", "").strip()
+
+    def _convert_value(self, v):
+        t = type(v)
+        if t in (list, int, float):
+            cv = v
+        elif t == str:
+            cv = self._sanitise_unicode(v)
+        elif t == bytes:
+            s = v.decode('ascii', 'replace')
+            cv = self._sanitise_unicode(s)
+        elif t == dicom.valuerep.DSfloat:
+            cv = float(v)
+        elif t == dicom.valuerep.IS:
+            cv = int(v)
+        elif t == dicom.valuerep.PersonName3:
+            cv = str(v)
+        else:
+            cv = repr(v)
+        return cv
+
+    def dicom_dataset_to_dict(self, ds):
```

Comment:
Hm, I think Django has some kind of idea about a serializer? Perhaps we should extend support via that instead of these custom routines? Alternatively, we could extend Django's JSON serializer? Anyway I think we should have this in ``utils.py`` or similar anyway :)

Reply:
Django Rest Framework Serializer: http://www.django-rest-framework.org/api-guide/serializers/